apisix/plugins/traffic-split.lua

```diff
@@ -203,6 +207,14 @@ local function new_rr_obj(weighted_upstreams)
         elseif upstream_obj.upstream then
             -- Add a virtual id field to uniquely identify the upstream key.
             upstream_obj.upstream.vid = i
+            if upstream_obj.upstream.nodes then
```

Comment:
Look like the nodes check here can be skipped?

Reply:
remove it